backend/models/__init__.py

```diff
@@ -1,13 +1,18 @@
-from .auth import *
+# isort:skip_file
```

Comment:
Why are we skipping the import sort?

Reply:
The `models` package has a specific order that the imports need to happen it.  `isort` will try to re-order the imports and break them.

Reply:
I thought that might be the case. Thanks for confirming!